lib/patches/config_patch.rb

```diff
@@ -11,3 +11,4 @@ def reload!
 end
 
 Config::Options.prepend(ConfigDecryptPasswords)
+Config::Options.include(MoreCoreExtensions::Shared::Nested)
```

Comment:
Wow, nice !!   Just tried fetch_path on Settings as well as Api::Settings with this PR. worked great.  👍